source/brailleDisplayDrivers/seikantk.py

```diff
@@ -145,10 +145,14 @@ def display(self, cells: List[int]):
 	def _onReceive(self, data: bytes):
 		"""
 		Note: Further insight into this function would be greatly appreciated.
-		- on each call, read three bytes only the second byte is used.
-		- first 3 bytes: command
-		- 1 byte: specify total length in bytes?
-		- variable length: arguments for command type
+
+		On each call, read three bytes only the second byte is used, and appended to a buffer.
```

Comment:
I still find "read three bytes only the second byte is used" hard to parse.

```suggestion
		On each call, read three bytes. Only the second byte is used, and appended to a buffer.
```

Reply:
I see, I've given this another look.